overlord/snapstate/snapstate.go

```diff
@@ -209,6 +209,63 @@ func Update(s *state.State, name, channel string, userID int, flags Flags) (*sta
 	return doInstall(s, snapst.Active, ss)
 }
 
+// Enable sets a snap to the active state
+func Enable(s *state.State, name string) (*state.TaskSet, error) {
+	if err := checkChangeConflict(s, name); err != nil {
```

Comment:
I would move the check down, no point is checking conflict if anyway it's enabled